tests/security/check_kernel_config/dm_crypt.pm

```diff
@@ -0,0 +1,61 @@
+# Copyright (C) 2021 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: dm crypt -> add flags to optionally bypass kcryptd
+#          workqueues, the options are 'no-read-workqueue' and
```

Comment:
It seems the options are 'no_read_workqueue' and 'no_write_workqueue'

Reply:
Thanks Lili, you are correct, let me fix it.